app/application/dto/order_dto.py

```
@@ -1,8 +1,7 @@
-from dataclasses import dataclass
+from pydantic import BaseModel
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)